CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing to oresat-firmware
+
+## General Information
+
+### Background
+
+OreSat is an oregon based nanosatellite project that is managed by the Portland State Aerospace Society (PSAS). We have one satellite currently in space, called OreSat0, learn more about it [here](https://www.pdx.edu/news/oregons-first-satellite-launched-orbit).
+
+Learn more about our organization [here](https://www.oresat.org/About-PSAS).
+
+### Geting involved
+
+The general guidelines for getting involved with OreSat can be found [here](https://www.oresat.org/get-involved).
+
+You can also follow our activity by taking a look at our [Google Calendar](https://calendar.google.com/calendar/u/0/embed?color=%23cca6ac&src=psas.rockets@gmail.com).
+
+## Getting started with OreSat Firmware development
+
+### Set up your development environment
+
+Follow [this](https://github.com/oresat/oresat-firmware#getting-started) to get your development environment setup.
+
+
+### Join a meeting
+
+Join one of our meetings. Meetings can be found in [Our Calendar](https://calendar.google.com/calendar/u/0/embed?color=%23cca6ac&src=psas.rockets@gmail.com). Look for "OreSat Firmware".
+
+### Development platforms
+
+  Here are links to some of our supported development boards.
+
+  * [NUCLEO-F091RC](https://www.mouser.com/ProductDetail/STMicroelectronics/NUCLEO-F091RC?qs=CE9ecZDhCK0jcTHFYd%2Fi5g%3D%3D#.YoAe6h4MCag.link)
+  * [NUCLEO-F446RE](https://www.mouser.com/ProductDetail/STMicroelectronics/NUCLEO-F446RE?qs=PRtH0mD6DWYnuBoPSlbRCA%3D%3D)
```

Comment:
This platform is no longer in use. C3 moved to the F439, and all the apps were updated to build for that rather than the F446, so I don't think anything will build for this anymore.